packages/application-shell/src/components/application-shell/application-shell.spec.js

```diff
@@ -654,14 +654,14 @@ describe('when clicking on navbar menu toggle', () => {
     await wait(() => {
       expect(window.localStorage.setItem).toHaveBeenCalledWith(
         STORAGE_KEYS.IS_FORCED_MENU_OPEN,
-        false
+        'false'
```

Comment:
The `localStorage.setItem` type signature expects a string

<img width="390" alt="image" src="https://user-images.githubusercontent.com/1110551/73127242-bb2ed980-3fbd-11ea-9fd2-d3d29a91d953.png">